### ISSUE TITLE
можете скинуть дамп базы данных в файле sql?

### ISSUE THREAD

**Issue by Nectorrr** [IvAlexsay/Test#1]

**Comment by IvAlexsay** [comment]
К сожалению, осталась только схема. Уверен, при необходимости, по ней можно восстановить базу
![схема](https://github.com/IvAlexsay/Test/assets/72877849/0c10efab-dc64-4d4d-ad2c-a772bfdb0f93)

**Comment by IvAlexsay** [comment]
Также в репозитории есть папка base_teststudents в которой лежит образ, по которому можно восстановить бд

**Comment by Nectorrr** [comment]
но схема не совсем соответствует образу в папке base_teststudents, т.е. я хочу спросить эта схема новой бд или старой

**Comment by Nectorrr** [comment]
нету таблиц test,result и т.д.

**Comment by IvAlexsay** [comment]
Есть

**Comment by Nectorrr** [comment]
я про схему

**Comment by IvAlexsay** [comment]
Да, вы правы, предыдущая схема, это ранняя версия, вот последняя
![image](https://github.com/IvAlexsay/Test/assets/72877849/73397bc5-2caa-4f55-99d9-0b8c20d2caa0)

**Comment by Nectorrr** [comment]
Здравствуйте, не подскажете как изменить файл auth.php чтобы в систему могло войти несколько студентов одновременно ,бьюсь второй день всё никак не могу понять, буду признателен если поможете
